crates/fuel-core/src/graphql_api/storage/da_compression.rs

```diff
@@ -35,6 +35,9 @@ pub mod reverse_key;
 pub mod script_code_codec;
 pub mod timestamps;
 
+#[cfg(feature = "fault-proving")]
+pub mod v2;
```

Comment:
Should this be named something more specific to the feature. I'm not sure I understand what `v2` means if it's always going to be hidden behind the `fault-proving` flag (I assume we are always going to keep it behind the flag because only the fault-proving nodes are going to want to track the extra state).

Perhaps we name it `fault-proving-extra-storage` or something specific like that?

Reply:
it won't always be behind the fault-proving flag :) we plan to remove the flag once the feature is done, but you're right, da-compression is enabled only on a few nodes.

Reply:
addressed in 4560027d74

I like the previous naming better though :/

Reply:
oh okay, will revert the commit with the name change then :)

Reply:
reverted in e54635b46a